### PR TITLE
Implement "nix fmt".

### PR DESCRIPTION
**Pull request by Lillecarl**
Adds support to run "nix fmt" within your nix configuration directory.

It'll use nixpkgs-fmt as the formatter, making sure your Nix code is indented and looks like "it's supposed to" 😄

### CODE REVIEW

**Comment by jlilja** [comment]
Sweet, I'll check this out soon!

File: flake.nix [hunk 1]
```diff
@@ -10,27 +10,30 @@
...
-        # this passes extra arguments into your modules, useful when you're a nix geek.
-        extraSpecialArgs = { inherit inputs; };
-        # import your modules
-        modules = [
-          ./home.nix
-        ];
+  outputs = { ... } @inputs:
+    let
+      pkgs = (import inputs.nixpkgs {
+        # allow unfree packages, we're not Richard Stallman
```
Comment (jlilja) [inline flake.nix:16]: Sad FSF noices

**Comment by jlilja** [comment]
How do I run the formatting for this pkg?

**Comment by Lillecarl** [comment]
"nix fmt" within the repo :)

**Pull request merged**